### PR TITLE
Add link to Limiting Run-once Pod Duration

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/2931

### CODE REVIEW

**Comment by mburke5678** [comment]
@jim-minter PTAL related to Link to Limiting Run-once Pod Duration for builds #2931

**Comment by mburke5678** [comment]
@wzheng1 Is this issue in your area? PTAL. Single sentence with a link added.

**Comment by mburke5678** [comment]
@wzheng1 PTAL Is this change in your area?

**Comment by wzheng1** [comment]
@mburke5678 Yes. QE will design test cases on this change.

**Comment by mburke5678** [comment]
@wzheng1 Any movement on this?

**Comment by wzheng1** [comment]
@mburke5678 sorry for the delay, here is the test case we created: https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-16408 and have tested in latest env, it works as expected.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL Minor change.

File: install_config/build_defaults_overrides.adoc [hunk 1]
```diff
@@ -28,19 +28,25 @@ allows administrators to change the global configuration at any time, and any
...
 currently supports overriding the `forcePull` flag on a build strategy to
 enforce always refreshing the local image during a build by pulling the image
 from the registry. This ensures that a user can only build with an image that
 they are allowed to pull. The plug-in can also be configured to apply a set of
 image labels to every built image.
++
+[NOTE]
+====
+You can define a hard limit on how long build pods can run by using the 
+xref:../admin_guide/managing_pods.adoc#manage-pods-limit-run-once-duration[RunOnceDuration plugin].
```
Comment (ahardin-rh) [inline install_config/build_defaults_overrides.adoc]: `RunOnceDuration` plug-in

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/build_defaults_overrides.adoc#install-config-build-defaults-overrides[Configuring Global Build Defaults and Overrides]
|Added a note on limiting how long pods can run and a link to xref:../admin_guide/managing_pods.adoc#manage-pods-limit-run-once-duration[RunOnceDuration plugin].
%

File: install_config/build_defaults_overrides.adoc (hunk 1 above)
Comment (mburke5678) [reply install_config/build_defaults_overrides.adoc]: @ahardin-rh The section I linked to uses *RunOnceDuration*. So, I went with that for consistency/

**Pull request merged**